### PR TITLE
my commit

### PR DESCRIPTION
**Pull request by Meng-Heng**

### CODE REVIEW

File: developer/language/reference/version.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+The `VERSION` statement was introduced in Keyman 3.0.
+
+The `&version` store was added in Keyman 5.0.
+
+Each subsequent version of Keyman updates the maximum store value
+accordingly to its major version.
+
+## See also
+
+-   [`&keyboardversion` store](keyboardversion.php)
```
Comment (jahorton) [inline developer/language/reference/version.md:61]: Looks like you were using an older version of the tool - it didn't clip off the `.php` extension here.

File: developer/language/reference/visualkeyboard.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+## Version history
+
+The `&visualkeyboard` store changed from .kvk to .kvks in Keyman 10.0.
+
+The `&visualkeyboard` store was added in Keyman 8.0.
+
+## See also
+
+-   [`&layoutfile` store](layoutfile.php)
```
Comment (jahorton) [inline developer/language/reference/visualkeyboard.md:65]: Another `.php` to trim off.

```suggestion
-   [`&layoutfile` store](layoutfile)
```

File: developer/language/reference/version.md (hunk 1 above)
Comment (jahorton) [reply developer/language/reference/version.md:61]:
```suggestion
-   [`&keyboardversion` store](keyboardversion)
```

**Review by Meng-Heng — COMMENTED** [review]
finished reviewing

**Review by jahorton — APPROVED** [review]

**Pull request merged**